lib/zendesk_apps_tools/settings.rb

```diff
@@ -57,8 +71,8 @@ def get_settings_from_file(filepath, parameters)
         end
 
         if !input && param['required']
-          puts "'#{param['name']}' is required but not specified in the config file.\n"
-          return nil
+          puts "\e[0;31m'#{param['name']}' is required but not specified in the config file.\e[0m\n"
```

Comment:
if not, you could set them to constants / local variables like we have [here](https://github.com/zendesk/docker-images/blob/master/dotfiles/.bashrc#L18-L26)